### PR TITLE
adding configurable options for root block device

### PR DESCRIPTION
**Pull request by jjduverge**

### CODE REVIEW

**Review by cdaniluk — CHANGES_REQUESTED** [review]
run `terraform fmt`, set `default = none` on `root_volume_size` so that it behaves as it did previously and sort your vars alphabetically

**Comment by jjduverge** [comment]
> 
I thought 'none' was not allowed and should be 'null'?

File: variables.tf [hunk 1]
```diff
@@ -79,6 +79,28 @@ variable "asg_subnets" {
   type        = list(string)
 }
 
+########################################
+# RootVolume Vars
+########################################
+
+variable "root_volume_encryption" {
+  default     = true
+  type        = boolean
```
Comment (sblack4) [inline variables.tf]:
```suggestion
  type        = bool
```

File: variables.tf [hunk 2]
```diff
@@ -79,6 +79,28 @@ variable "asg_subnets" {
...
 }
 
+########################################
+# RootVolume Vars
+########################################
+
+variable "root_volume_encryption" {
+  default     = true
+  description = "Encrypted root volume"
+  type        = boolean
```
Comment (sblack4) [inline variables.tf]:
```suggestion
  type        = bool
```

**Review by sblack4 — CHANGES_REQUESTED** [review]

**Review by cdaniluk — APPROVED** [review]

**Pull request merged**